File: templates/matlab/observer_estimation_polar_signature.m
function fig = observer_estimation_polar_signature()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('polar_profile', 1710, 'observer and state estimation: polar signature', 'observer and state estimation', 'polar signature');
end

File: templates/python/observer_estimation_polar_signature.py
"""observer_estimation_polar_signature: 观测器与状态估计极坐标指纹（polar 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='observer and state estimation: polar signature'):
    return make_template_figure('polar_profile', seed=1710, title=title, domain='observer and state estimation', topic='polar signature')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
